eng/common/scripts/Verify-Readme.ps1

```diff
@@ -0,0 +1,15 @@
+# Wrapper Script for Readme Verification
```

Comment:
While the script is the core of the work we could have also included a Devop yml template that captures the call to this script as well.

Reply:
It seems that that will be mostly redundant because there is not much that the yml step will add, We can just add the powershell step directlry to the pipeline that calls docwarden.